sample_apps/multidomain_simple_app/script

```diff
@@ -1,236 +1,397 @@
 	--policy_key_output_file=client_policy_key_file.bin \
 	--policy_cert_output_file=client_policy_cert_file.bin \
 	--platform_key_output_file=platform_key_file.bin \
 	--attest_key_output_file=attest_key_file.bin
 
-$CERTIFIER_PROTOTYPE/utilities/cert_utility.exe --operation=generate-policy-key-and-test-keys \
+$CERTIFIER_PROTOTYPE/utilities/cert_utility.exe \
+    --operation=generate-policy-key-and-test-keys \
 	--policy_key_name=server-policy-key \
 	--policy_key_output_file=server_policy_key_file.bin \
 	--policy_cert_output_file=server_policy_cert_file.bin \
 	--platform_key_output_file=platform_key_file.bin \
 	--attest_key_output_file=attest_key_file.bin
 
-
+# ---------------------------------------------------------------------------
+# Implemented in gen_policy_key_for_example_app()
+# ---------------------------------------------------------------------------
 # embed policy key and compile app
 cd $MULTIDOMAIN_EXAMPLE_DIR/provisioning
 $CERTIFIER_PROTOTYPE/utilities/embed_policy_key.exe \
 	--input=client_policy_cert_file.bin \
 	--output=../client_policy_key.cc
+
 $CERTIFIER_PROTOTYPE/utilities/embed_policy_key.exe \
 	--input=server_policy_cert_file.bin \
 	--output=../server_policy_key.cc
 
 # make app and compute app measurement
 cd $MULTIDOMAIN_EXAMPLE_DIR
-make -f multidomain_app.mak
+make -j "${NumMakeThreads}" -f multidomain_app.mak
 
 # build policy
 cd $MULTIDOMAIN_EXAMPLE_DIR/provisioning
 
-#policyKey says platformKey is-trused-for-attestation
+# ---------------------------------------------------------------------------
+# policyKey says platformKey is-trusted-for-attestation
 
 $CERTIFIER_PROTOTYPE/utilities/make_unary_vse_clause.exe \
 	--key_subject=platform_key_file.bin \
-	--verb="is-trusted-for-attestation" --output=ts1.bin
+	--verb="is-trusted-for-attestation" \
+    --output=ts1.bin
 
-# vse_policy1a.bin is server-policy-key says platform-key is-trusted-for-attestation
+# server_vse_policy1a.bin is server-policy-key says platform-key is-trusted-for-attestation
 $CERTIFIER_PROTOTYPE/utilities/make_indirect_vse_clause.exe \
 	--key_subject=server_policy_key_file.bin \
-	--verb="says" --clause=ts1.bin --output=vse_policy1a.bin
-# vse_policy1b.bin is client-policy-key says platform-key is-trusted-for-attestation
+	--verb="says" \
+    --clause=ts1.bin \
+    --output=server_vse_policy1a.bin
+
+# client_vse_policy1b.bin is client-policy-key says platform-key is-trusted-for-attestation
 $CERTIFIER_PROTOTYPE/utilities/make_indirect_vse_clause.exe \
 	--key_subject=client_policy_key_file.bin \
-	--verb="says" --clause=ts1.bin --output=vse_policy1b.bin
+	--verb="says" \
+    --clause=ts1.bin \
+    --output=client_vse_policy1b.bin
 
+# Up to here done in construct_policyKey_platform_is_trusted_multidomain_simple_app()
+
+# ---------------------------------------------------------------------------
+# Implemented in produce_signed_claims_for_vse_policy_statement_multidomain_simple_app()
```

Comment:
For future reference & maintenance, I annotated most steps in this script with the fn-name in the `run_example.sh` driver script that implements these commands.